dsa_actionkit/dsa_actionkit/settings.py

```diff
@@ -0,0 +1,150 @@
+"""
+Django settings for dsa_actionkit project.
+
+Generated by 'django-admin startproject' using Django 3.2.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+import os
+from pathlib import Path
+
+DEBUG = True
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ["DJANGO_SECRET_KEY"]
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    "mydsa.apps.MyDsaConfig",
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+]
+
+MIDDLEWARE = [
+    "django.middleware.security.SecurityMiddleware",
+    "django.contrib.sessions.middleware.SessionMiddleware",
+    "django.middleware.common.CommonMiddleware",
+    "django.middleware.csrf.CsrfViewMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+    "django.middleware.clickjacking.XFrameOptionsMiddleware",
+]
+
+ROOT_URLCONF = "dsa_actionkit.urls"
+
+WSGI_APPLICATION = "dsa_actionkit.wsgi.application"
+
+
+# Database
+# https://docs.djangoproject.com/en/3.2/ref/settings/#databases
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.mysql",
+        "NAME": "ak_roboticdogs",
+        "USER": "USERNAME",
+        "PASSWORD": "ROBOTIC_DOGS_SECRET_KEY",
+        "HOST": "roboticdogs.client-db.actionkit.com",
+    }
```

Comment:
can we load these from a .env file instead of keeping a placeholder value in git?

Reply:
yeah i was thinking we could move to that (noted in the readme) if there's some creds we can use that aren't tied to a personal account? (the only creds i have are my own) unless that's the only kind there is.
if it's something each person would need to set up individually then i would need instructions on how to set that up, and then I could add it into the readme as well

Reply:
Enabled .env file imports in my follow-up PR.  By default, the project will use the SQLite database. If/when necessary, users can connect to robotic dogs by including their credentials in a .env file and running the [appropriate command](https://docs.djangoproject.com/en/5.0/topics/db/multi-db/)